.changeset/seven-swans-glow.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/plugin-tech-insights': minor
```

Comment:
Think all of these should be `patch` changes as there's no breaking changes and all the packages are 0.x for now 👍

Reply:
Done